salt/modules/x509.py

```diff
@@ -637,7 +645,7 @@ def write_pem(text, path, pem_type=None):
     return 'PEM written to {0}'.format(path)
 
 
-def create_private_key(path=None, text=False, bits=2048):
+def create_private_key(path=None, text=False, bits=2048, verbose=False):
```

Comment:
I think we likely will want to preserve current functionality by setting versbose to True as a default. That way we won't surprise people during upgrades, but the option is there to turn it off.